### PR TITLE
MAINT: Update .mailmap

### PR DESCRIPTION
**Pull request by charris**
Contributor nevimov did not want his full name used.

[ci skip]

### CODE REVIEW

**Comment by charris** [comment]
Let's not merge this just yet, there may be more modifications needed befor the 1.12 branch.

File: .mailmap [hunk 1]
```diff
@@ -18,7 +18,6 @@ Alex Griffing <argriffi@ncsu.edu> alex <argriffi@ncsu.edu>
 Alex Griffing <argriffi@ncsu.edu> argriffing <argriffi@ncsu.edu>
```
Comment (madphysicist) [inline .mailmap]: This should probably be removed too.

Comment (madphysicist) [inline .mailmap]: In fact all entries of the form "A <B> A <B>" and "A <B> <B>" should probably be removed.

Comment (charris) [inline .mailmap]: Why? It associates a real name to the nickname.

Comment (charris) [inline .mailmap]: Yep, looks like you are right. Entries of the form "A <B> <B>" are valid though, remaps possible nicknames based on address. Just found some decent .mailmap documentation at https://git-scm.com/docs/git-shortlog.

Comment (charris) [inline .mailmap]: Looks like some others, also. I'm not going to worry about that now but do a cleanup sometime in the future.

**Pull request merged**